lib/active_model/serializer/adapter/json.rb

```diff
@@ -7,37 +7,13 @@ class Json < Adapter
         def serializable_hash(options = nil)
           options ||= {}
           if serializer.respond_to?(:each)
-            @result = serializer.map { |s| FlattenJson.new(s).serializable_hash(options) }
```

Comment:
re `@result`.... at least until https://github.com/rails-api/active_model_serializers/pull/1117 is merged.. it's bad OO in any case, for sure.